exec/src/klio_exec/commands/run.py

```diff
@@ -59,8 +59,8 @@
 # NOTE: hopefully we don't get an dict lookup errors since KlioConfig
```

Comment:
We'd get a dict lookup error if the user sets `pipeline_options.streaming` to `True` but sets `job_config.events.inputs[].type` to something other than `pubsub`. `KlioConfig` _should_ catch that (and does, but in an inelegant/uphelpful way), but since we're doing a [dict lookup](https://github.com/spotify/klio/blob/develop/exec/src/klio_exec/commands/run.py#L503) rather than a `dict.get`, there will always be that possibility of a dict lookup error. We do a dict lookup because it's easier - it will automatically raise for us (granted, unhelpful), rather than us having to do a `dict.get("foo", None); if foo raise Exception`.